pkgs/servers/tailscale/default.nix

```diff
@@ -35,6 +34,6 @@ buildGoModule rec {
     homepage = "https://tailscale.com";
     description = "The node agent for Tailscale, a mesh VPN built on WireGuard";
     license = licenses.bsd3;
-    maintainers = with maintainers; [ danderson mbaillie twitchyliquid64 ];
+    maintainers = with maintainers; [ danderson mbaillie twitchyliquid64 jk ];
```

Comment:
Not objecting particularly, but curious why you want the burden :)

Reply:
Tailscale is pretty handy so im happy to keep it updated.

I also keep an eye on headscale :)